Harsh_mnc_sub/Assignment/Assignment-1.ipynb

```diff
@@ -0,0 +1,3452 @@
+      "\n",
+      "\n",
+      "\n",
+      "Valerie: I remember how different became dangerous.\n",
+      "\n",
+      "\n",
+      "\n",
+      "Evey: No one will ever forget that night and what it meant for this country. But I will never forget the man and what he meant to me.\n",
+      "\n",
+      "\n",
+      "\n",
+      "Deitrich: You wear a mask for so long, you forget who you were beneath it.\n",
+      "\n",
+      "\n",
+      "\n",
+      "Finch: The problem is, he knows us better than we know ourselves.\n",
+      "\n",
+      "\n",
+      "\n",
+      "Valerie: But what I hope most of all is that you understand what I mean when I tell you that, even though I do not know you, and even though I may never meet you, laugh with you, cry with you, or kiss you, I love you. With all my heart, I love you.\n",
+      "\n",
+      "\n",
+      "\n",
+      "V: What was done to me was monstrous.\n",
+      "\n",
+      "Evey: And they created a monster.\n",
+      "\n",
+      "\n",
+      "\n",
+      "Evey: God is in the rain.\n",
+      "\n",
+      "\n",
+      "\n",
+      "V: Your powers of observation continue to serve you well.\n",
+      "\n",
+      "\n",
+      "\n",
+      "Evey: He was Edmond Dantes. And he was my father, and my mother, my brother, my friend. He was you, and me. He was all of us.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "dialogues = open('dialogues.txt')\n",
+    "for i in dialogues:\n",
+    "    print(i)\n",
+    "#dialogues.read()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Here a dialogue is seperated from the speaker by a colon :\n",
+    "\n",
+    "for example\n",
+    "\n",
+    "<b>Delia Surridge</b>: Is it meaningless to apologize?<br>\n",
+    "<b>V</b>: Never.<br>\n",
+    "<b>Delia Surridge</b>: I'm so sorry.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### There are 2 tasks\n",
+    "#### 1)To find out unique actors and store them in a seperate array called speakers\n",
+    "#### 2)Then you have to create a dict called labels that maps every speaker to a list of it's dialogues"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The required outputs are :-\n",
+    "####    Task1\n",
+    "\n",
+    "####  `speakers=['V','Evey' ,....]`\n",
+    "\n",
+    "###    Task2\n",
+    "\n",
+    "#### `labels=`\n",
+    "#### `{\"V\":[\" A man after my own heart.\", ..]`,\n",
+    "#### `\"Evey\": [\"Are you like a crazy person?\",...],..}`\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 74,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['V', '\\n', 'E', 'D', 'A', 'C', 'L', 'O', 'F']\n"
```

Comment:
As a first look. You are having only the first characters of speakers iguess. Split from the colon and add the entire list